src/main/scala/io/iohk/ethereum/vm/package.scala

```diff
@@ -5,6 +5,6 @@ package object vm {
   /**
     * Number of 32-byte UInt256s required to hold n bytes (~= math.ceil(n / 32))
     */
-  def wordsForBytes(n: UInt256): BigInt =
-    if (n.isZero) 0 else (n - 1) / UInt256.Size + 1
+  def wordsForBytes(n: BigInt): BigInt =
+    if (n.compare(0) == 0) 0 else (n - 1) / UInt256.Size + 1
```

Comment:
In `n.compare(0)` the argument is still converted to a `BigInt`, so I don't see how that's better than the natural `n == 0`